packages/app/src/cli/commands/app/web-env.ts

```diff
@@ -0,0 +1,37 @@
+import {appFlags} from '../../flags.js'
+import {AppInterface} from '../../models/app/app.js'
+import {load as loadApp} from '../../models/app/loader.js'
+import {webEnv} from '../../services/web-env.js'
+import {Flags} from '@oclif/core'
+import {output, path, cli} from '@shopify/cli-kit'
+import Command from '@shopify/cli-kit/node/base-command'
+
+export default class WebEnv extends Command {
```

Comment:
This command should be hidden. I think `static hidden = true` will do the job. If not hidden then we have to get the naming etc. right!